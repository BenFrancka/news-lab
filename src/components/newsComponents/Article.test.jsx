import dotenv from 'dotenv';
dotenv.config();
import React from 'react';
import { render, screen } from '@testing-library/react';
import Article from './Article';

describe('Article', () => {
  it.skip('should be a single article object', async () => {
    const article = (
      <>
        <h2>title</h2>
        <h3>author</h3>
        <p>description</p>
      </>
    );
    render(<Article article={article} />);

    const h2 = await screen.findByText('title');
    const h3 = await screen.findByText('author');
    const p = await screen.findByText('description');

    expect(h3).toMatchSnapshot();
    expect(h2).toMatchSnapshot();
    expect(p).toMatchSnapshot();
  });
});
