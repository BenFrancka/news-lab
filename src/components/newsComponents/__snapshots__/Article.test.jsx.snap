// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Article should be a single article object 1`] = `
<h3>
  author
</h3>
`;

exports[`Article should be a single article object 2`] = `
<h2>
  title
</h2>
`;

exports[`Article should be a single article object 3`] = `
<p>
  description
</p>
`;
